src/EditorFeatures/TestUtilities/CodeActions/AbstractCodeActionTest.cs

```diff
@@ -105,5 +112,24 @@ protected static Document GetDocument(TestWorkspace workspace)
         {
             return workspace.CurrentSolution.GetDocument(workspace.Documents.First().Id);
         }
+
+        private class TestPickMembersService : IPickMembersService
+        {
+            private readonly ImmutableArray<string> _memberNames;
+
+            public TestPickMembersService(ImmutableArray<string> memberNames)
+                => _memberNames = memberNames;
+
+            public PickMembersResult PickMembers(string title, ImmutableArray<ISymbol> members)
+                => new PickMembersResult(_memberNames.SelectAsArray(n => members.Single(m => m.Name == n)));
+        }
+
+        protected Task TestWithGenerateConstructorDialogAsync(
```

Comment:
This gets refactored in the GenerateEqualsAndGetHashCode tests in the next PR.  The idea is that this is the helper function that allows us to test the feature while swapping in a mock IPickMembersService.